src/cr/cube/stripe.py

```diff
@@ -465,12 +518,16 @@ def count(self):
 
     @lazyproperty
     def pruned(self):
-        """True if this row is "empty"."""
+        """True if this question was answered by exactly zero respondents."""
         return self._unweighted_count == 0 or np.isnan(self._unweighted_count)
 
     @lazyproperty
     def table_proportions(self):
-        """float between 0.0 and 1.0 quantifying this row's share of stripe total."""
+        """float between 0.0 and 1.0 quantifying this row's share of stripe total.
```

Comment:
Could we decide on either using the arrow `"->"` or the combination of `"between"` and `"and"` for consistency?